account_fiscal_position_rule/test/test_rules.yml

```diff
@@ -65,6 +55,7 @@
       'partner_id': ref('base.res_partner_11'),
       'partner_invoice_id': ref('base.res_partner_11')
     }
+    self.env['res.partner'].browse(ref('base.main_company')).write({'country_id': ref('base.us')})
```

Comment:
Do browse res.partner with ref('base.main_company')? the correct reference should be ref('base.main_partner')?